app/sync.js

```diff
@@ -282,26 +290,39 @@ module.exports.init = function (initialState) {
         newDeviceId ? new Immutable.List(newDeviceId) : null)
     }
   })
-  if (getSetting(settings.SYNC_ENABLED) !== true) {
-    return
-  }
-  ipcMain.on(messages.GET_INIT_DATA, (e) => {
-    const seed = initialState.seed || null
-    deviceId = initialState.deviceId || null
-    e.sender.send(messages.GOT_INIT_DATA, seed, deviceId, config)
-  })
   ipcMain.on(messages.SYNC_READY, module.exports.onSyncReady.bind(null,
-    !initialState.seed && !initialState.deviceId))
+    hasRun ? false : !initialState.seed && !initialState.deviceId))
```

Comment:
minor: it's a little hard to read this ternary within a function call; might be cleaner to assign a variable right before.